jacodb-analysis/src/main/kotlin/org/jacodb/analysis/codegen/Generator.kt

```diff
@@ -197,8 +197,7 @@ private fun runCmd(
     try {
         val cmdBuilder = ProcessBuilder()
         cmdBuilder.directory(workingDir).command(cmd)
-        val cmdProcess = cmdBuilder.redirectError(errorFile).redirectOutput(outputFile).start()
-        cmdProcess.waitFor()
+        cmdBuilder.redirectError(errorFile).redirectOutput(outputFile).start().waitFor()
```

Comment:
you can call this chain right after `ProcessBuilder()`. Make clean separation between 
1. process builder initialization
2. process starting
3. process waiting